.github/PULL_REQUEST_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,69 @@
+<!--- Use this only if you've created new ACE component (manager, util, etc.). Please provide a name of your newly created ACE component in title of pull request. -->
+
+## Description
+<!--- Describe your changes in few sentences. If your pull request is related to an issue, please put `Closes #issueNumber` or `Related to #issueNumber` sentence here. -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## ACE inclusion self-assesment
+<!--- It's most likely that we'll include your code in ACE's codebase anyway, however please take a few minutes to answer following questions, which will allow making more conscious decision. -->
+
+**Are there significantly different solutions to same problem? If so, is it the most common/generic one?**
+
+<!--- Please answer here -->
+
+**Are other solutions better and should ACE incorporate it instead of this change in the long run?**
+
+<!--- Please answer here -->
+
+**Is given functionality needed or nice to have for most of games? Is it used often in tandem with ACE?**
+
+<!--- Please answer here -->
+
+**Is it small/simple enough to not require much maintenance?**
+
+<!--- Please answer here -->
+
+**Is the functionality related to a core Amiga concept?**
+
+<!--- Please answer here -->
+
+**Is it entirely, or its ACE interface, written by core ACE team or frequent contributors?**
+
+<!--- Please answer here -->
+
+**Is it a building block or dependency for other parts of code? How much coupling there is between this part and rest of ACE code?**
+
+<!--- Please answer here -->
+
+**Does it allow making significantly better games at reasonable performance cost?**
+
+<!--- Please answer here -->
+
+**Is most of the code first-party?**
+
+<!--- Please answer here -->
+
+**Does it require additions/changes/removal of some other parts of ACE functionality/tooling for its use?**
+
+<!--- Please answer here -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Especially point out in which productions it was used so far -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
```

Comment:
If this template is aimed for new components... How idea of new component maps on bug fixes / breaking changes?

Reply:
we don't really have a good record of keeping PRs atomic. ;) I think oftentimes components will come with some changes to core code - be it refinements or bugfixes.